docs/05-e2e-testing.md

```
@@ -0,0 +1,118 @@
+---
+
+## Step 1: Start Keycloak Using Docker
+
+Follow the official Keycloak Docker guide to set up Keycloak:
+
+```bash
+docker run -p 8080:8080 -e KC_BOOTSTRAP_ADMIN_USERNAME=admin -e KC_BOOTSTRAP_ADMIN_PASSWORD=admin quay.io/keycloak/keycloak:26.1.4 start-dev
+```
+
+Access Keycloak Admin Console:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)